src/host/polyfills/simpleView.ts

```diff
@@ -114,6 +114,15 @@ export function applyStylesToggleFocusPatch(content: string): string | null {
     return replaceInSourceCode(content, pattern, replacementText);
 }
 
+export function applyNoMatchingStylesPatch(content: string): string | null {
+    // Patch to inform user to refresh/resume target to get CSS information when attaching to a paused target.
+    const pattern = /this\._noMatchesElement\.textContent\s*=\s*(ls\s*`No matching selector or style`);/g;
+    const replacementText = `
+       this._noMatchesElement.innerHTML = ls \`No matching selector or style.<br />Styles may not be available if target was paused when opening Edge DevTools.<br />Please resume or refresh the target.\`;
```

Comment:
@vidorteg I'm curious what your thoughts are on this particular patch when it comes to Loc. I know this will need to be updated for Loc V2, but I'm also curious how we are going to get updated Loc files for patches when we update strings like this

Reply:
Thanks for bringing this up. I made a note on the https://github.com/microsoft/vscode-edge-devtools/issues/341 to include this case in the localization task.

Reply:
should not be a blocker for now as the content meet the bar by being localizable.